README.md

```diff
@@ -0,0 +1,47 @@
+# Visualization of configuration pushes
+
+This internship project is about a service internal to Google that is responsible for dynamically pushing pieces of configuration or data to running processes.
+
+The two main questions this projected tried to address are:
```

Comment:
```suggestion
The two main questions this project tried to address are:
```

Reply:
I used the past tense because we write this at the end of the project. I changed it to what you suggested in case the the text you will add will match better.